src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public interface Drawable {
+    String TEXT_COLOR = "color: ";
+    String TEXT_UNITS = " units, ";
+    String TEXT_SQ_UNITS = " sq. units, ";
+    String TEXT_AREA = "area: ";
```

Comment:
I thought that that constants for draw method, so they will fit more in Drawable instead of Figure

Reply:
> I don't think if in the future we want Car to be Drawable it would need all these constants. It's more about Figure

I thought that that constants for draw method, so they will fit more in Drawable instead of Figure